src/lib/pubkey/ec_group/ec_apoint.h

```diff
@@ -78,16 +82,16 @@ class BOTAN_UNSTABLE_API EC_AffinePoint final {
                                              std::span<const uint8_t> domain_sep);
 
       /// Multiply a point by a scalar returning a complete point
-      ///
-      /// Workspace argument is transitional
-      EC_AffinePoint mul(const EC_Scalar& scalar, RandomNumberGenerator& rng, std::vector<BigInt>& ws) const;
+      EC_AffinePoint mul(const EC_Scalar& scalar, RandomNumberGenerator& rng) const;
+
+      BOTAN_DEPRECATED("Use version without workspace arg")
+      EC_AffinePoint mul(const EC_Scalar& scalar, RandomNumberGenerator& rng, std::vector<BigInt>& ws) const {
+         BOTAN_UNUSED(ws);
+         return this->mul(scalar, rng);
+      }
 
       /// Multiply a point by a scalar, returning the byte encoding of the x coordinate only
-      ///
-      /// Workspace argument is transitional
-      secure_vector<uint8_t> mul_x_only(const EC_Scalar& scalar,
-                                        RandomNumberGenerator& rng,
-                                        std::vector<BigInt>& ws) const;
+      secure_vector<uint8_t> mul_x_only(const EC_Scalar& scalar, RandomNumberGenerator& rng) const;
```

Comment:
For the record: This API [was added recently](https://github.com/randombit/botan/pull/4503) and hence doesn't require a deprecation, correct?

Reply:
Correct.

If this ends up not making it for 3.7 I'll switch it to be like the others with a second deprecated function that ignores the arg

Reply:
Changed so as to not break SemVer